docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,27 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes filter versions that will not run on JDK 8.
+This issue is fixed in the latest versions: 
+
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v3.1.12-plugins-filters-date.html[date filter 3.1.12]
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v7.2.7-plugins-filters-geoip.html[geoip filter 7.2.7]
+
+You can <<updating-plugins,update these plugins>> as usual:
+
+[source,sh]
+-----
+bin/logstash-plugin update logstash-filter-date
+bin/logstash-plugin update logstash-filter-geoip
+-----
+
+TIP: Clear the `JAVA_HOME` environment variable to run Logstash with the bundled JDK. 
```

Comment:
Given that we do support setting custom JDKs via `LS_JAVA_HOME` env var in `7.16`, WDYT about including it here for completeness?

Reply:
I would consider mentioning it without going into a lot of detail.  We're getting pretty far away from the original known issue.  Would that approach alleviate your concerns?

Reply:
Yes, just something like clear the `JAVA_HOME` and `LS_JAVA_HOME` env vars?

Reply:
Messaging this is a bit tricky, because we really don't want users overriding the bundled version unless they have a really good reason.  I'll see what I can come up with.

Reply:
I like the "tip" messaging here. It is quick and concise. Including "JDK 11" may help tie it together.

```suggestion
TIP: Clear the `JAVA_HOME` environment variable to run Logstash with the bundled JDK 11. 
```

OR

```suggestion
TIP: Clear the `JAVA_HOME` and `LS_JAVA_HOME` environment variables to run Logstash with the bundled JDK 11. 
```

Reply:
I had implemented yet another variation before I saw your suggestions.  Lemme know what you think.